include/Game/GameEvents.hpp

```diff
@@ -1 +1,13 @@
+#ifndef _game_events_hpp
+#define _game_events_hpp
 
+enum GameEvents {
```

Comment:
nit picky, but considering how this cannot be an enum class...you may want to wrap this in a namespace.